7.0 add update date

lepistone opened the pull request.

lepistone committed:
initial commit for stock_picking_update_date

It is green already, but I need to handle the stock.picking.in and out
and add some docs in __openerp__.py

lepistone committed:
allow to update the date on all picking types

lepistone committed:
update date: improve description

pedrobaeza commented on stock_picking_update_date/model/picking.py:
```diff
@@ -0,0 +1,139 @@
+
+class Picking(orm.Model):
+
+    _name = 'stock.picking'
```
This is not needed, because the name is the same as inherited.

pedrobaeza commented on stock_picking_update_date/model/picking.py:
```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
```
Please name it _stock_picking.py_

bwrsandman replied:
Adding to that, please only put one class/model per file and name the class and file after the model.

pedrobaeza commented on stock_picking_update_date/model/picking.py:
```diff
@@ -0,0 +1,139 @@
+
+class PickingIn(orm.Model):
+
+    _name = 'stock.picking.in'
```
This is not needed.

pedrobaeza commented on stock_picking_update_date/model/picking.py:
```diff
@@ -0,0 +1,139 @@
+
+class PickingOut(orm.Model):
+
+    _name = 'stock.picking.out'
```
This is not needed

pedrobaeza commented:
Hi, @lepistone, thanks for the PR.

Can you explain why the standard field cannot be used for this purpose?

Regards.

lepistone committed:
rename things as suggested

lepistone committed:
improve comments and tooltips

lepistone commented:
Thanks for reviewing @pedrobaeza and @bwrsandman. I addressed your remarks. I didn't split the classes in separate files, because they are really small and related and the resulting file is still very short.

pedrobaeza commented:
:+1: 

Thanks

rvalyi commented:
:+1:  LGTM

gurneyalex commented:
:+1:

guewen commented:
:+1:
